front_end/src/reducers/TileReducer.js

```diff
@@ -0,0 +1,49 @@
+import {
+  REQUEST_ADD_TILE,
+  RECEIVE_ADD_TILE,
+  REQUEST_TILES,
+  RECEIVE_TILES,
+  REQUEST_TILE,
+  RECEIVE_TILE
+} from 'actions/Tiles/TileActions';
+
+const initialState = {
+  rows: [],
+  details: [],
```

Comment:
I updated so it should read `details: {}`. I think you're supposed to avoid using null as an initial state anyway since that is the point.